src/app/server/app.go

```diff
@@ -60,51 +68,68 @@ func NewApp(opts ...AppOptions) *App {
 		),
 	}
 
-	// Define routes and middlewares
-	app.Engine.StaticFS("/static", &assetfs.AssetFS{
-		Asset:    Asset,
-		AssetDir: AssetDir,
-		Prefix:   "static",
-	})
+	// Define static dir
+	app.Engine.Static("/static", "/client")
 
-	// Load embedded templates
-	app.Engine.SetHTMLTemplate(
-		binhtml.New(Asset, AssetDir).MustLoadDirectory("templates"),
-	)
+	// Load embedded templates MISSING
+	app.Engine.SetRenderer(echoRenderer{})
 
 	// Map app struct to access from request handlers
 	// and middlewares
-	app.Engine.Use(func(c *gin.Context) {
+	var contextSetterMiddleware func(c *echo.Context) error
+	contextSetterMiddleware = func(c *echo.Context) error {
 		c.Set("app", app)
-	})
+		return nil
+	}
+
+	app.Engine.Use(contextSetterMiddleware)
+
+	var requestIDMiddleware func(c *echo.Context) error
+	requestIDMiddleware = func(c *echo.Context) error {
+		id, _ := uuid.NewV4()
+		c.Set("uuid", id)
+		return nil
+	}
+
+	// Map uuid for every requests
+	app.Engine.Use(requestIDMiddleware)
 
 	// Avoid favicon react handling
-	app.Engine.GET("/favicon.ico", func(c *gin.Context) {
+	app.Engine.Get("/favicon.ico", func(c *echo.Context) error {
 		c.Redirect(301, "/static/images/favicon.ico")
+		return nil
+	})
+
+	//Force handle index to React
+	app.Engine.Get("/", func(c *echo.Context) error {
```

Comment:
I don't know why I could not get index to work, so I did that quick workaround but it not something intended to be the final release.